releases/v1.4.1.toml

```diff
@@ -0,0 +1,35 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.4.0"
+
+pre_release = false
+
+preface = """\
+The first patch release for `containerd` 1.4 includes a fix for v1 shims hanging
+on exit and exec when the log pipe fills up along with other minor changes.
+
+### Notable Updates
+
+* Always consume shim logs to prevent logs in the shim from blocking [containerd/containerd#4546](https://github.com/containerd/containerd/pull/4546)
+* Update CNI Dependencies [containerd/containerd#4555](https://github.com/containerd/containerd/pull/4555)
```

Comment:
nit add blurb:  `CNI v0.8.0 and CNI plugins v0.8.6 have been tested to work with this service update.`

Reply:
Actually probably just remove this section as I don't see a specific reason it was backported in the first place. Just didn't quite make the cutoff. The dependencies changes should show up at the bottom, I'll see if I can get that fixed.